src/benchmarks/micro/corefx/System.Threading.Tasks.Dataflow/Perf.TransformManyBlock.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Threading.Tasks.Dataflow.Tests
+{
+    public class TransformManyBlockPerfTests : ReceivablePropagatorPerfTests<TransformManyBlock<int, int>, int>
+    {
+        public override TransformManyBlock<int, int> CreateBlock() =>
+            new TransformManyBlock<int, int>(
+                i => new int[] { i },
+                new ExecutionDataflowBlockOptions
+                {
+                    MaxDegreeOfParallelism = Environment.ProcessorCount
```

Comment:
Why do other blocks test for non-parallel and this one only tests parallel?

Reply:
Good remark! Fixed.